DirectX11/HackerDXGI.cpp

```diff
@@ -1891,6 +2061,38 @@ STDMETHODIMP HackerDXGISwapChain::SetFullscreenState(THIS_
 	LogInfo("  Fullscreen = %d\n", Fullscreen);
 	LogInfo("  Target = %p\n", pTarget);
 
+	HRESULT hr;
+
+	if (mFakeBackBuffer || mFakeSwapChain) // we need fullscreen anyway
+	{
+		BOOL fst = FALSE;
+		IDXGIOutput *target = nullptr;
+		mOrigSwapChain->GetFullscreenState(&fst, &target);
+
+		if (target)
+			target->Release();
+
+		// dont call setfullscreenstate again to avoid starting mode switching and flooding winproc with unnecessary messages
+		// can disable fullscreen mode somehow
+		if (fst && Fullscreen)
```

Comment:
OK.  If we run into games where it proves to be a problem, we can revisit it then.